_develop/collections.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Work with Collections (Preview)
+{:.no_toc}
+
+Collections is the feature designed to make data portable among Blockstack applications. Sharing is accomplished by storing a user's data in a standard, structured way in a known Gaia location. Collections associate the data with a user's distributed ID. When users move among apps, the same data is available to each application. 
```

Comment:
Does this sound better?
"Sharing is accomplished by storing a user's data in a standardized format in a known location."

Also I don't think we've used the term "distributed ID" before.

Reply:
I don't think that change is significant either way. I do think it is important to mention Gaia.  I'll switch distributed ID for decentralized throughout.